### ISSUE TITLE
Can't load old transaction history (> 8 Apr)

### ISSUE THREAD

**Issue by churik** [status-im/status-react#10771]
# Bug Report

## Problem
Reproducible only on mainnet.
If open transaction history and tap "Load more history" more than once - no more history is loaded. Therefore, for my account I'm unable to load history older than 8 Apr.

#### Expected behavior
More history is loaded

#### Actual behavior
nothing happens

#### Notes

For the developer who will be investigating. Is it related to another PR? Tips on where to start? 

#### Acceptance Criteria

Rules for the future PR to be accepted.

### Reproduction

[comment]: # (Describe how we can replicate the bug step by step.)
**Prerequisites: account has long transaction history**

- Open Status
- Go to wallet
- Open Status account
- Tap "Load more history"
- Tap "Load more history" one more time

### Additional Information

- Status version: release 1.4
- Operating System: Android, iOS

#### Logs

Full logs: 
[status_logs-3.zip](https://github.com/status-im/status-react/files/4745341/status_logs-3.zip)

```
06-08 12:07:22.599 13609 13853 D StatusModule: Signal event: {"type":"wallet","event":{"type":"recent-history-ready","blockNumber":10224463,"accounts":["0xcf2272205cc0cf96cfbb9dd740bd681d1e86901e"],"newTransactions":null,"erc20":false}}
06-08 12:07:23.640 13609 13923 D StatusModule: Signal event: {"type":"wallet","event":{"type":"newblock","blockNumber":10224464,"accounts":[],"newTransactions":{},"erc20":false}}
06-08 12:07:32.332 13609 13929 D StatusModule: Signal event: {"type":"discovery.stopped","event":null}

```

**Comment by cammellos** [comment]
Is this still an issue @churik ?

**Comment by churik** [comment]
Yes, still an issue on my account.
@Serhy can you recheck please?
![BAACAgQAAxkBAAEsfQhfjtY7WBAyZXJyRPEvtaxipZAQdAACAg0AAlB3cVA1qJ7DcgilohsE_1603196511 43](https://user-images.githubusercontent.com/4557972/96585403-da099480-12df-11eb-943f-503974c52d39.gif)
